tests/Aether.Tests/Aether.Testing.fs

```diff
@@ -0,0 +1,212 @@
+﻿module Aether.Testing
+
+open FsCheck
+open Swensen.Unquote
+
+/// Properties that can be used to evaluate the conformance of Lens, Prism, and Isos
+/// to certain invariants.
+module Properties =
+    /// Lens properties
+    [<RequireQualifiedAccess;CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+    module Lens =
```

Comment:
I've added self-contained re-definitions of `get`/`set`/`map` into this file and removed the import of `Aether`. This allows the file to be even more portable and imported even in libraries that don't want to bring in a dependency on Aether itself.